src/overlays/effects/ovl_Effect_Ss_D_Fire/z_eff_ss_d_fire.h

```diff
@@ -6,12 +6,13 @@
 typedef struct {
     /* 0x00 */ Vec3f pos;
     /* 0x0C */ Vec3f velocity;
-    /* 0x18 */ Vec3f accel;
-    /* 0x24 */ s16 scale;
-    /* 0x26 */ s16 scaleStep;
-    /* 0x28 */ s16 alpha;
-    /* 0x2A */ s16 fadeDelay;
-    /* 0x2C */ s32 life;
+    /* 0x14 */ Vec3f accel;
+    /* 0x20 */ s16 scale;
+    /* 0x22 */ s16 scaleStep;
+    /* 0x24 */ s16 alpha;
+    /* 0x26 */ s16 fadeDelay;
+    /* 0x28 */ s16 life;
+    /* 0x2C */ s32 unk_2C;
 } EffectSsDFireInitParams; // size = 0x30
```

Comment:
The offsets look wrong
```suggestion
    /* 0x18 */ Vec3f accel;
    /* 0x24 */ s16 scale;
    /* 0x26 */ s16 scaleStep;
    /* 0x28 */ s16 alpha;
    /* 0x2A */ s16 fadeDelay;
    /* 0x2C */ s16 life;
    /* 0x30 */ s32 unk_30;
} EffectSsDFireInitParams; // size = 0x34
```